xapian-core/queryparser/queryparser.lt

```diff
@@ -477,8 +477,8 @@ static unsigned int yy_find_shift_action(
       if( iLookAhead<sizeof(yyFallback)/sizeof(yyFallback[0])
              && (iFallback = yyFallback[iLookAhead])!=0 ){
         LOGLINE(QUERYPARSER,
-		"FALLBACK " << ParseTokenName(iLookAhead) << " => " <<
-		ParseTokenName(iFallback));
+    "FALLBACK " << ParseTokenName(iLookAhead) << " => " <<
+    ParseTokenName(iFallback));
```

Comment:
Here (and in several places below) your patch changes the indentation of lines you haven't otherwise modified.

If there are whitespace fixes to be made, it is better to make them as a separate commit rather than mixing them with functional changes.  In this case, these changes are not useful, so please revert them.  It makes for a smoother review process if you make such changes as "fixup" commits - see https://xapian-developer-guide.readthedocs.io/en/latest/contributing/contributing-changes.html#open-a-pull-request-on-github

Reply:
The reason for the indentation issues is because earlier I had set my settings to: use tabs and tab size to 4 spaces. After asking James in IRC and checking HACKING, I had adjusted the settings accordingly (8 spaces). And as I started had afresh after deleting the fork, there are no fixup commits in #186.